docs/unified-test-documentation/dasharo-compatibility/315-network-boot.md

```diff
@@ -234,3 +234,72 @@ server.
 1. The iPXE application boots successfully.
 1. iPXE obtains an IP address.
 1. iPXE boots an `Debian 11` from netboot.xyz.
+
+## USB008.001 Enable Network Boot (Firmware)
+
+**Test description**
+
+This test aims to verify if Network Boot can be enabled.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `SETUP_MENU_KEY` = F2
```

Comment:
Is this the correct setup key for ALL Dasharo platforms?

Reply:
Any setup isn't valid for all Dasharo platforms. In that case, should I remove this information?

Reply:
I think yes. In my opinion we should prepare the table with BIOS and Setup menu keys and add it to the documentation (but not in this PR).